templates/vue/src/services/TapestryAPI.js

```diff
@@ -58,6 +58,12 @@ class TapestryApi {
     }
   }
 
+  async exportTapestry() {
+    var url = `${apiUrl}/tapestries/${this.postId}/export`
```

Comment:
I think a better name for this would be `getTapestryExport` since the function itself isn't doing the "exporting".

Also let's avoid using `var` for declarations and use `const` or `let` instead. Keeps things consistent.